README.md

```diff
@@ -1,19 +1,16 @@
 [![Build status](https://badge.buildkite.com/c76805e51e194fb0cdf4bf537306e3b6270cb1ebc4db48f21c.svg)](https://buildkite.com/monadic/radicle-link)
 
-# Radicle Link
+# Radicle Link 🌱
 
-Second iteration of the Radicle code collaboration protocol and stack.
+This is the working repo for the second iteration of the [Radicle](http://radicle.xyz/) code collaboration protocol and stack.
 
-**WORK IN PROGRESS**
+**🚨 WORK IN PROGRESS 🚨**
 
-_In fact, there is nothing substantial to see here, yet_
+_In fact, there is nothing substantial to see here yet_
 
-# Contact
+Join our [#radicle](irc://freenode:1/radicle) IRC channel for development updates. 
```

Comment:
```suggestion
Join our [#radicle](https://webchat.freenode.net/?channels=radicle) IRC channel for development updates. 
```

(`irc://` links don't seem to render, even if valid)

Reply:
sounds good - thanks for the check